lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -101,6 +109,13 @@ def self.available_options
             default_value: true,
             type: Boolean
           ),
+          FastlaneCore::ConfigItem.new(
+            key: :skip_if_exists,
+            description: 'If the file already exists in the S3 bucket, just log the error instead of failing with `user_error!`',
```

Comment:
Nitpick: "the error" is not introduced in the sentence before. What do you think of

```suggestion
            description: 'If receiving an error from S3 because the file already exists in the bucket, just log the error instead of failing with `user_error!`',
```

Not the best, to be honest. It's challenging to be succinct while also informative in these short descriptions. So, happy for this to be discarded if it doesn't seem like a clear improvement.

Reply:
Actually S3 doesn't return an error: it's not like we try to upload and check if S3 rejected us. What we do is we do a HEAD request first to check if the file exists (checking if it returned a content size > 0 or not), and then do the upload only if it doesn't. So mentioning "if S3 returned an error" here would be misleading I think.

Reply:
@mokagio Improved in dd723fa. using this wording instead:

> If the file already exists in the S3 bucket, skip the upload (and report it in the logs), instead of failing with `user_error!`

Let me know what you think 🙂

Reply:
:shipit: